crates/sui-benchmark/src/workloads/workload.rs

```diff
@@ -12,7 +12,8 @@ use crate::ValidatorProxy;
 
 // This is the maximum gas we will transfer from primary coin into any gas coin
 // for running the benchmark
-pub const MAX_GAS_FOR_TESTING: u64 = 10_000_000_000_000;
+pub const MAX_GAS_FOR_TESTING: u64 = 100_000_000_000;
+pub const MAX_BUDGET_FOR_TESTING: u64 = 50_000_000_000;
```

Comment:
What is `MAX_BUDGET_FOR_TESTING` for?

Reply:
It is the max budget that any one transaction can have of 50 SUI